templates/partials/filters/date.html

```diff
@@ -1,5 +1,5 @@
-<span id="dateRangeDropdown" class="filter dropdown mm-dropdown">
-  <span id="date-filter" class="mm-button">
+<span id="dateRangeDropdown" class="filter dropdown mm-dropdown" ng-class="{ 'disabled': selectMode && selected.length }">
```

Comment:
I think the disabling would make more sense in `reports_fitlers.html`.
And you could do them all in the same file, which would be nice.